Wikipedia/Code/EditNoticesViewController.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+protocol EditNoticesViewControllerDelegate: AnyObject {
+    func editNoticesControllerUserTapped(url: URL)
+}
+
+class EditNoticesViewController: ViewController {
```

Comment:
Mostly just did this out of inertia and to minimize having to add boilerplate surrounding initialization and theme management to the controller. If it still makes you uncomfortable though, let me know and I can refactor it.

Reply:
Yeah if you don't mind - we have a `ThemeableViewController` that you could subclass, it might save you a little bit of code on theming. You'll be able to remove lines 33-34 as well. Those are probably harmless but would be nice to remove the `scrollView` assignment if it won't be used. After some tinkering looks like this init change works:

```
    init(theme: Theme, viewModel: EditNoticesViewModel) {
        self.viewModel = viewModel
        super.init(nibName: nil, bundle: nil)
        self.theme = theme
    }
```

Reply:
> Yeah if you don't mind - we have a `ThemeableViewController` that you could subclass

@tonisevener No problem – done!